contracts/Distributor.sol

```diff
@@ -183,8 +183,8 @@ contract Distributor is UUPSHelper {
             address token = tokens[i];
             uint256 amount = amounts[i];
 
-            // Checking if only an approved operator can claim for `user`
-            if (onlyOperatorCanClaim[user] == 1 && operators[user][msg.sender] == 0) revert NotWhitelisted();
+            // Only approved operator can claim for `user`
+            if (msg.sender != user && operators[user][msg.sender] == 0) revert NotWhitelisted();
```

Comment:
So `onlyOperatorCanClaim` is useless

Reply:
Maybe we can inverse by not letting anyone claiming by default and if toggle anybody can claim